plugins/missions.js

```diff
@@ -219,7 +344,7 @@ window.plugin.missions = {
 		}).map(function(waypoint) {
 			return [waypoint.portal.latE6/1E6, waypoint.portal.lngE6/1E6];
 		});
-		
+
```

Comment:
Please remove deletion of blank lines. This and other lines below. If you want to reform the code (according to eslint rules) this should be done in a separate PR for all files

Reply:
> Please remove deletion of blank lines.

...and other white-space-only changes. At least in separate commit.
Otherwise you make reviewing meaningful changes harder.

Reply:
Whitespaces are back. Sorry for the multiple commits. I am not used to git. Still using subversion at work.